test-apps/storage-unit-test-app/storage-unit-test-app.gradle

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+apply plugin: "com.android.application"
+
+android {
+  compileSdkVersion project.targetSdkVersion
+
+  defaultConfig {
+    applicationId "com.example.flubber"
+    minSdkVersion project.minSdkVersion
+    targetSdkVersion project.targetSdkVersion
+    versionCode 1
+    versionName "1.0"
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    multiDexEnabled true
+  }
+
+  sourceSets {
+    main {
+      java {
+        srcDir "../../firebase-storage/src/testUtil/java"
+      }
+    }
+  }
+
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+}
+
+dependencies {
+  // We intentionally use an open ended version to pick up any SNAPSHOT
+  // versions published to the root project' s build/ directory.
+  implementation 'com.google.firebase:firebase-storage:16+'
+  implementation('com.google.firebase:firebase-core:16+') {
+    exclude group: 'com.google.firebase', module: 'firebase-common'
+  }
+
+  implementation "com.android.support:design:27.1.1"
+  implementation "com.android.support:appcompat-v7:27.1.1"
+
+  androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+  androidTestImplementation 'com.android.support.test:rules:1.0.2'
+}
+
+
+// ==========================================================================
+// Copy from here down if you want to use the google-services plugin in your
+// androidTest integration tests.
+// ==========================================================================
+ext.packageName = "com.example.flubber"
+apply from: '../../gradle/googleServices.gradle'
```

Comment:
This is not needed, google-services.json is automatically handled for smoke tests, see `test-apps/build.gradle` for hot it happens.